### DISCUSSION TITLE
management web interface

### DISCUSSION THREAD

**Discussion by fifisoft57** [hnimminh/libresbc#35] [General]
hello I just installed Libresbc (with a little difficulty being not used to ansible) on your presentation there is a management web interface, where can you find this page?

thank you

**Comment by hnimminh** [comment]
> hello I just installed Libresbc (with a little difficulty being not used to ansible) on your presentation there is a management web interface, where can you find this page?
> 
> 
> 
> thank you

The web ui is in todo list, and I still not have resource/time to work on this yet.

**Reply by camglegg** [reply]
Is this something you are looking for help with? I would love to contribute.

**Reply by hnimminh** [reply]
Yup, any contribute are welcome
